metaflow/plugins/argo/argo_workflows_cli.py

```diff
@@ -123,7 +124,13 @@ def argo_workflows(obj, name=None):
     "--only-json",
     is_flag=True,
     default=False,
-    help="Only print out JSON sent to Argo Workflows. Do not deploy anything.",
+    help="Only print out JSON sent to Argo Workflows. Do not deploy anything. DEPRECATED, prefer --output-yaml instead",
```

Comment:
Any reason to mark this as deprecated? We haven't deprecated any functionality in Metaflow this far in the project's history

Reply:
Mainly because it's "broken" in that it doesn't output _all_ objects and my thought was to have `--only-yaml` replace it. But I can remove this note.

Reply:
Removed.